### ISSUE TITLE
solve the UNSOLVABLE TIME PROBLEM 

### ISSUE THREAD

**Issue by hannahedeland** [EXMARaLDA/exmaralda#293]
- plain/direct TEI export can contain UNSOLVABLE TIME PROBLEM structures, but since there is no time (as in `<anchor>`s)  involved, just `<seg>`s, there is no check and no problem
- WebLicht result TEI files are currently normalized before they are written to file, normalize.xsl should catch the problem and not insert `<anchor>`s into `<seg>`s if the `<anchor>`s can't be sorted, but if we want to normalize these files, then we should probably rather warn before the processing starts if TEI or re-import is chosen, since processing can take a while and that would be frustrating
- or should these structures maybe simply not be allowed/recommended in any kind of ISO/TEI exported from the P-E, maybe caught by the structure errors check as a "structure warning"? Do we need (to support) them for a use case i can't think of? 
- Extended unsolvable example based on AnneWill, exported as HIAT ISO/TEI:
```
[...]
         <annotationBlock who="SPK0" start="T37" end="T45" xml:id="au3">
            <u xml:id="u3">
               <seg xml:id="seg6" type="utterance" subtype="declarative">
                  <w xml:id="w217">Ich</w>
                  <w xml:id="w218">will</w>
                  <anchor synch="T38" />
                  <w xml:id="w219">kurz</w>
                  <w xml:id="w220">hier</w>
                  <w xml:id="w221">bleiben</w>
               </seg>
               <seg xml:id="seg7" type="utterance" subtype="declarative">
                  <w xml:id="w222">Genau</w>
                  <w xml:id="w223">hier</w>
                  <w xml:id="w224">möchte</w>
                  <w xml:id="w225">ich</w>
                  <w xml:id="w226">bei</w>
                  <w xml:id="w227">der</w>
                  <anchor synch="T39" />
                  <w xml:id="w228">Wahl<anchor synch="T40" />rechtsä<anchor synch="T41" />nderung</w>
                  <w xml:id="w229">bleiben</w>
               </seg>
               <anchor synch="T42" />
[...]
         <annotationBlock who="SPK1" start="T38" end="T39" xml:id="au16">
            <u xml:id="u16">
               <seg xml:id="seg31" type="utterance" subtype="interrogative">
                  <w xml:id="w244">Was</w>
               </seg>
               <seg xml:id="seg32" type="utterance" subtype="interrogative">
                  <w xml:id="w245">Was</w>
                  <w xml:id="w246">sagen</w>
                  <w xml:id="w247">Sie</w>
                  <w xml:id="w248">jetzt</w>
               </seg>
            </u>
         </annotationBlock>
[...]
```

**Comment by berndmoos** [comment]
@hannahedeland : What's the status of this one? We should either resolve it or declare it a non-issue...

**Comment by hannahedeland** [comment]
I think I'm not the one to answer the questions I put in the comment :) We would need some kind of decision on how to handle these cases, and maybe we should include a check (before WebLicht processing or anything else including normalization starts), or just a note, or we could just ignore everything including AnneWill...

**Comment by berndmoos** [comment]
These structures come about when utterances boundaries (in HIAT) do not coincide with event boundaries. This should not happen when transcribers follow the HIAT conventions. This cannot be solved in TEI (it is... : unsolvable!). Instead we could

1. Check if it is really mentioned in the HIAT manual
2. Provide an additional "structure check" in the PE - under "Segmentation Errors"?

Note that there are solvable and unsolvable time problems. Solvable are missing timepoint in monologuous stretches. Unsolvable in overlapping stretches!

**Comment by hannahedeland** [comment]
Update:
- There is no HIAT utterance boundary requires event boundary convention in the manual, but a warning as an addition to segmentation errors would be helpful: #305
- The cases of "UNSOLVABLE TIME PROBLEM" that are actually solvable should be handled by the fix in #288  
- The few cases that are really not solvable can wait until a more distant milestone

**Comment by berndmoos** [comment]
Closing this, it is too unsolvable
